src/main/java/fact/features/singlePulse/timeSeriesExtraction/SinglePulseExtractor.java

```diff
@@ -46,10 +46,28 @@ public int numberOfPulses() {
         }
 
         public double timeSeriesBaseLine() {
-            DescriptiveStatistics statistics = 
+            DescriptiveStatistics statistics =
                 new DescriptiveStatistics(timeSeriesAfterExtraction);
             return statistics.getMean();
-        }    
+        }
+
+        /**
+         * @param: startSlice is with respect to extraction window start.
```

Comment:
This javadocstring is not correct. See http://www.oracle.com/technetwork/articles/java/index-137868.html